views/dashboard.ejs

```diff
@@ -2,42 +2,27 @@
 <%- include('./partials/nav'); %>
 
 <div class="dashboard-container">
-
 	<div class="page-header">
-  		<h1>Dashboard</h1>
-		<% riders.forEach(function(riders) { %>
-  		<p id="dashb-name">Hi <%= riders.name %>!</p>
-			<p id="dashb-balance">Your points balance is: <%= riders.pointBalance %></p>
-		<% }) %>
-	</div>
-
-	<div class="row">
-		<h2>Redeem Your Points - Link Version</h2>
-	  <p>Displays all Rewards; links to redeem page to see each point displayed individually</p>
-	  <ul>
-	  <% rewards.forEach(function(reward) { %>
-	    <li><a href="/dashboard/<%= reward.id %>"><%= reward.rewardType %></a>:</li>
-	  <% }) %>
-	  </ul>
+		<h1>Dashboard</h1>
+		<p id="dashb-name">Hi <%= rider.name %>!</p>
+		<p id="dashb-balance">Your points balance is: <%= rider.pointBalance %></p>
 	</div>
 
-	<h2>Redeem Your Points - Form Version</h2>
-	<p>Displays all Rewards in a dropdown menu; SHOULD redirect to redeem page when "submit" button is clicked. But when this happens, no data is displayed</p>
 	<form action="/redeem" method="post">
-	  <div class="form-group">
-			<input type="hidden" name="" value="">
-	  </div>
+	<div class="form-group">
+	  <input type="hidden" name="pointBalance" value="<%= rider.pointBalance %>">
+	</div>
 
-  <div class="form-group">
-    <label for="reward">Reward</label>
-    <select name="rewardType" id="points">
-			<option value="choice" selected>Choose Your Reward</option>
-      <% rewards.forEach(function(reward){ %>
-        <option value="<%= reward.id %>"><%= reward.rewardType %>: <%= reward.pointsRequired %> Points</option>
-      <% }) %>
-    </select>
-  </div>
-  <input type="submit" class="btn btn-primary">
-</form>
+	<div class="form-group">
+	<label for="reward">Reward</label>
+	<select name="pointsRequired" id="pointsRequired">
```

Comment:
@tinuola  Generally, you  only pass one piece of data using a select.  If you need multiple pieces of data 1) don't use select; use multiple inputs including hidden inputs.  2) if you use select,  pass back one piece of data that can be used to find a record in the database.